.github/workflows/lint.yml

```diff
@@ -0,0 +1,107 @@
+name: Lint
+
+on:
+  pull_request:
+
+jobs:
+  # Determine what the js changes are
+  js-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      packages: ${{ steps.filter.outputs.changes }}
+    steps:
+      # Use paths-filter to determine which paths have changes
+      # https://github.com/dorny/paths-filter
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          # Format of key: value. When there is a change detected in value, the
+          # key will be returned in the output.
+          filters: |
+            packages/dashboard: packages/dashboard/**
+            packages/dashboard-v2: packages/dashboard-v2/**
+            packages/dnslink-api: packages/dnslink-api/**
+            packages/handshake-api: packages/handshake-api/**
+            packages/health-check: packages/health-check/**
+            packages/website: packages/website/**
+
+  # Determine what the dockerfile changes are
+  dockerfile-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      packages: ${{ steps.filter.outputs.changes }}
+    steps:
+      # Use paths-filter to determine which paths have changes
+      # https://github.com/dorny/paths-filter
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          # Format of key: value. When there is a change detected in value, the
+          # key will be returned in the output.
+          filters: |
+            docker/nginx/Dockerfile: docker/nginx/Dockerfile
+            docker/sia/Dockerfile: docker/sia/Dockerfile
+            packages/dashboard/Dockerfile: packages/dashboard/Dockerfile
+            packages/dnslink-api/Dockerfile: packages/dnslink-api/Dockerfile
+            packages/handshake-api/Dockerfile: packages/handshake-api/Dockerfile
+            packages/health-check/Dockerfile: packages/health-check/Dockerfile
+            packages/website/Dockerfile: packages/website/Dockerfile
```

Comment:
Can we refactor this to
```
dockerfile:
 - '**/Dockerfile'
```
per Karol's example above?

Reply:
no because the hadolint accepts the path to the dockerfile.